volatility3/framework/constants/linux/__init__.py

```diff
@@ -11,6 +11,14 @@
 
 """The value hard coded from the Linux Kernel (hence not extracted from the layer itself)"""
 
+# Translation Layer constants
+PAGE_BIT_PRESENT = 0
+PAGE_BIT_PSE = 7  # Page Size Extension: 4 MB (or 2MB) page
+PAGE_BIT_PROTNONE = 8
```

Comment:
These don't belong in linux unless they're linux specific constants (in which case we need to rethink how they're used based on which OS the layer is constructed on.

Reply:
done